staging/src/k8s.io/apiserver/pkg/audit/request.go

```diff
@@ -195,11 +195,7 @@ func LogResponseObject(ctx context.Context, obj runtime.Object, gv schema.GroupV
 	}
 	if status, ok := obj.(*metav1.Status); ok {
 		// selectively copy the bounded fields.
-		ae.ResponseStatus = &metav1.Status{
-			Status: status.Status,
-			Reason: status.Reason,
-			Code:   status.Code,
-		}
+		ae.ResponseStatus = status
```

Comment:
I took a closer look at what's included in a `metav1.Status`, and I think there's some stuff we'd want to filter out. Notably, the `ListMeta` is just going to be noise that bloats the logs. I'd suggest falling back to the previous approach, something like:

```suggestion
		ae.ResponseStatus = &metav1.Status{
			Status: status.Status,
			Message: status.Message,
			Reason: status.Reason,
			Details: status.Details,
			Code:   status.Code,
		}
```

I don't have an opinion on whether to include details.

Reply:
@tallclair please take a look again when you get the chance please